### ISSUE TITLE
Long-term support and roadmap

### ISSUE THREAD

**Issue by benediktdertinger** [OllieWP/ollie#72]
Hi @mikemcalister,

thank you for this amazing theme and all the work and thoughts you put into this product. Working many years with all kinds of CMS and following along the changes (Gutenberg, FSE etc.) the Wordpress core team took in the past years i have to admit: It has been a very bumpy road and we had to spent many hours fixing things (like a loooot!!).

As we mostly rely on React, NodeJS, headless etc. today in our projects and started to use tools like Webflow etc. because of its simplicity and ease of use (as well as reduced costs in maintenance etc.) we almost fully abandoned Wordpress and its plugins except for a few shop systems. I already saw your answer to this [Woocommerce related question](https://github.com/OllieWP/ollie/issues/55). 

It is the first time in many years i saw a piece of code in the WP themes / plugin ecosystem that actually looks well-built and very promising. So thank you a lot for making this publicly available!

We are now at the point were we want to move some of our last projects from WP to other framworks but are actually thinking about sticking with WP if there is an actual perspective for projects like yours.

TL;DR – My questions
1. What are your plans in terms of long-term support?
2. Are you open for external contributions?
3. Do you have a roadmap for your theme?
4. What are the next components, patterns and extensions you are planning to integrate?
5. Is there anything missing at the moment new adoperts should be aware of?
6. Are there any essential downsides or problems you have been facing with using the new theme structure and coding patterns?
7. With all your expertise: What is your perspective in terms of the ecosystem and new features in WP and Gutenberg /FSE (i know this is quite generic, but maybe you have one or two lines for us)?

We know how much effort it takes to build something like this and highly appreciate what you built and shared here!

<3

**Comment by mikemcalister** [comment]
Hey Benedikt, 

First off, thanks for taking the time and leaving a thoughtful comment here! I'm glad people are enjoying the theme, it's been really encouraging. 

1. Ollie is now my full-time focus. I'll be continuing to grow out the free version as well as a pro version with added features. Right now, we're working with the reviewers on WordPress.org to get this theme approved for the official repository. That means you'd be able to install and update it via the WordPress interface. We're hoping to be live in the next week or two, but things over there are all volunteer run, so it really depends on how quick that moves. 
2. Definitely open to contributions, fixes, suggestions, etc. Fire away. 
3. I don't have a public roadmap yet. The focus will be on more patterns, more blocks (probably pro blocks) to do some more advanced designs/features. We're also toying with the idea of a Ollie Cloud, a way of working pushing and pulling patterns/assets to a cloud. Once we get the theme live publicly, we can work on a more formalized roadmap. 
4. See above. Free patterns, pro patterns, blocks and pro blocks. 
5. I wouldn't say there's anything missing from Ollie, per se. It just ties into the new WordPress features/interface. And as more features get added, we'll continue to support those. 
6. I would say the biggest hurdle for most folks is the time investment it takes to learn all this new stuff. There's a lot! But it's powerful. It's different and sometimes maddening, but it's going to catch on, I'm very confident in that. You basically have to treat it like learning any other new tool. It takes time, but you learn about all the quirks and all the hidden tips and tricks. It gets easier to the point where it's kind of crazy how fast you can build in there. The [6.4 roadmap just dropped](https://make.wordpress.org/core/2023/08/22/roadmap-to-6-4/), and it's going to blow people away. 6.3 was a huge release too, but 6.4 is going to be huge. 
7. The product market has been slow to respond. I'm not exactly sure why. When these features were introduced, there was a lot of "Well, let's wait and see." But now these features ship in core, and it's enabled on every single new site, so there's no waiting and seeing, it's here! It's hard to build products on shaky ground, I get that. That's why we're just finally going live with Ollie — WordPress is ready. I think the block and block theme code was a bit too far outside the wheelhouse of the standard WP dev. And agency folks haven't been incentivized because it takes a lot of time/effort/money to switch workflows, and it hasn't been obvious if the payoff was going to be worth it. In my opinion, it is going to be worth it. The same big companies that use WordPress now are going to continue using it, and slowly adopting this new paradigm because it is objectively better, in my opinion. You just have to figure out how to wield it in a way that works for you. 

Hopefully that's helpful! We're just two guys building this thing, so we're doing what we can with the resources we have. We're excited to see folks reaching out with these kinds of questions, because they're important. I'm fairly active on [my Twitter](https://twitter.com/mikemcalister) and the [Ollie Twitter](https://twitter.com/buildwithollie) (or X, whatever) if you want to follow along there. I'm also publishing video tutorials to help folks learn this stuff [over on YouTube](https://www.youtube.com/@OllieWP).

See ya! ✌️

**Comment by ValLynn** [comment]
Thank you Mike.

 Val
